### PR TITLE
Make prompt more consistent

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

File: core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java [hunk 1]
```diff
@@ -16,9 +16,9 @@
 
     private static final Logger log = Logger.getLogger("io.quarkus.test");
 
-    public static final String DISABLED_PROMPT = "\u001b[33mTests Disabled, press [e] to enable\u001b[0m";
+    public static final String PAUSED_PROMPT = "\u001b[33mTests pause, press [r] to resume\u001b[0m";
```
Comment (gsmet) [inline core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java:19]:
```suggestion
    public static final String PAUSED_PROMPT = "\u001b[33mTests paused, press [r] to resume\u001b[0m";
```

File: core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java [hunk 2]
```diff
@@ -37,7 +37,7 @@ public void install() {
         public void handleInput(int[] keys) {
             if (disabled) {
```
Comment (gsmet) [inline core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java:38]: Should it be changed to paused?

File: core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java [hunk 3]
```diff
@@ -130,7 +130,7 @@ public void testsEnabled() {
     @Override
     public void testsDisabled() {
```
Comment (gsmet) [inline core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java:131]: Same here?

**Review by gsmet — CHANGES_REQUESTED** [review]
Added a few comments.

File: core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java (hunk 2 above)
Comment (stuartwdouglas) [reply core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java:38]: Maybe, but there is a different internal 'paused' that happens when a dev mode hot reload is going on (so you don't have the tests potentially slowing down user interaction).

**Review by gsmet — APPROVED** [review]

**Pull request merged**